sympy/physics/quantum/density.py

```diff
@@ -2,11 +2,13 @@
 
 from itertools import product
 
-from sympy import Tuple, Add, Mul, Matrix, log, expand, Rational
+from sympy import Tuple, Add, Mul, Matrix, log, expand, sqrt, Rational
 from sympy.core.trace import Tr
+from sympy.core.compatibility import u
+from sympy.mpmath import acos, pi
```

Comment:
We've dropped mpmath from our tree, you should change these to just `from sympy import acos, pi`